specification/quota/resource-manager/Microsoft.Quota/stable/2025-09-01/groupquota.json

```diff
@@ -0,0 +1,2276 @@
+            }
+          ]
+        }
+      }
+    },
+    "GroupQuotasEntity": {
+      "type": "object",
+      "description": "Properties and filters for ShareQuota. The request parameter is optional, if there are no filters specified.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "title": "Properties",
+              "type": "object",
+              "allOf": [
+                {
+                  "$ref": "#/definitions/GroupQuotasEntityBase"
+                }
+              ]
+            }
+          }
+        }
+      ]
+    },
+    "GroupQuotaList": {
+      "type": "object",
+      "description": "List of Group Quotas at MG level.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Group Quotas at MG level.",
+          "items": {
+            "$ref": "#/definitions/GroupQuotasEntity"
+          }
+        },
+        "nextLink": {
+          "description": "The URL to use for getting the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "GroupQuotasEnforcementListResponse": {
+      "type": "object",
+      "description": "List of Azure regions, where the group quotas is enabled for enforcement.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Azure Regions.",
+          "items": {
+            "$ref": "#/definitions/GroupQuotasEnforcementStatus"
+          }
+        },
+        "nextLink": {
+          "description": "The URL to use for getting the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "EnforcementState": {
+      "description": "Enforcement status.",
+      "enum": [
+        "Enabled",
+        "Disabled",
+        "NotAvailable"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "EnforcementState",
+        "modelAsString": true
+      }
+    },
+    "GroupQuotasEnforcementStatus": {
+      "type": "object",
+      "description": "The GroupQuota Enforcement status for a Azure Location/Region.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "properties": {
+            "enforcementEnabled": {
+              "$ref": "#/definitions/EnforcementState",
+              "description": "Is the GroupQuota Enforcement enabled for the Azure region."
+            },
+            "enforcedGroupName": {
+              "type": "string",
+              "description": "The name of the group that is enforced.",
```

Comment:
>description [](http://example.com/codeflow?start=15&length=11)

Is this an enum, or its user specified?
What are the constraints? What validations do you perform on its value?